### ISSUE TITLE
Few accessibility issues (RM #4935)

### ISSUE THREAD

**Issue by dpage** [postgres/pgadmin4#3119]
***Issue migrated from Redmine: https://redmine.postgresql.org/issues/4935***
*Originally created by **Murtuza Zabuawala** at 2019-11-14 12:55:08 UTC.*

1) Suppress ESC key presses in Alertify dialogues when the come from Select2 controls (It also closes the alertify dialog)
2) Do not allow tabindex focus on Switch control when it is disabled
3) Tab keyboard navigation does not work in wcDocker iframe

**Comment by dpage** [comment]
Issue closed on Redmine.
